core-container/entry.sh

```diff
@@ -178,6 +178,10 @@ function run_init() {
     echo "--< Starting init >--"
     wait_kcp_ready
     kubestellar init --local-kcp false --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW
+    run_dex
```

Comment:
Don't you need to make sure it's running as you do it in the background ?

Reply:
Yes, added check for it.